tests/test_local_links.py

```diff
@@ -0,0 +1,87 @@
+import re
+from datetime import datetime
+from glob import glob
+from pathlib import Path
+from urllib.parse import urlparse
+
+import pytest
+import pytz
+from bs4 import BeautifulSoup
+from freezegun import freeze_time
+
+from lib.alerts import Alerts
+
+
+def get_page_html(env, template_path, alert_data=None):
+    template = env.get_template(template_path)
+    if alert_data is not None:
+        content = template.render({'alert_data': alert_data})
+    else:
+        content = template.render()
+    return BeautifulSoup(content, 'html.parser')
+
+
+def get_local_route_from_template_path(template_path):
+    local_route = '/alerts/' + Path(template_path).stem
```

Comment:
what does `.stem` do? :3

Reply:
It give you the filename with its suffix (see https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.stem). I was using a RegExp to do the same but see that we use `Path` in other places so thought this would be easier.

Reply:
You mean without the suffix, right? At least that's what the docs say. Thanks for the link, I now get it ☺️

Reply:
I do! 😬